koku/api/common/permissions/hcs_access.py

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright 2023 Red Hat Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Defines the Settings Access Permissions class."""
+from rest_framework import permissions
+
+
+class HCSAccessPermission(permissions.BasePermission):
+    """Determines if a user can post HCS data."""
+
+    def has_permission(self, request, view):
+        """Check permission to view and post HCS report data."""
+        if request.user.admin:
```

Comment:
Does this mean only and Org Admin user can Post?

Reply:
Great question, I can't say I have actually tested it strictly but I believe that would be the case based on how others work. Since this is being posted potentially by a Lambda function we probably need to change this. :sweat_smile:

Reply:
I do think absent a true service account, we probably want to keep this open to anyone with a login for now.